mkdocs/cli.py

```diff
@@ -120,13 +123,15 @@ def json_command(clean, config_file, strict, site_dir):
 @cli.command(name="gh-deploy")
 @click.option('--clean', is_flag=True, help=clean_help)
 @click.option('--config-file', type=click.File('rb'), help=config_file_help)
-def gh_deploy_command(config_file, clean):
+@click.option('--message', '-m', help=commit_message_help)
+@click.option('--branch', '-b', help=remote_branch_help)
```

Comment:
Call it `--remote-branch` to make it clear and that's the flag we have in the current version that got lost in a recent refactor.